### ISSUE TITLE
Getting  [Errno 13] Permission denied    |    autotrain.cli.run_dreambooth:run:393 - Job ID: 33508    |    autotrain.trainers.common:wrapper:121 - [Errno 13] Permission denied:

### ISSUE THREAD

**Issue by SakshiKhadilkar** [huggingface/autotrain-advanced#632]
I setup autotrain-advanced on my local machine Windows11 with 12GB GPU Memory

Run the following commands 

1. pip install -U autotrain-advanced 
2. autotrain setup --update-torch 
3. autotrain dreambooth --train  --model stabilityai/stable-diffusion-xl-base-1.0  --image-path images  --prompt "photo of Narendra Modi" --resolution 1024 --batch-size 1 --num-steps 500  --gradient-accumulation 4 --lr 1e-4  --project-name "text-to-image" --mixed-precision fp16

The images on which I'm training my text-to-image model is present under the 'images' folder in current directory.

@abhishekkrthakur  I followed your YouTube Tutorial  
https://youtu.be/gF078Lhnr94?si=JDbFwW9FLjbFhX5h

Getting this errors, pls help me resolve this 

![image](https://github.com/huggingface/autotrain-advanced/assets/117538804/0ffdb9be-a391-437b-a06c-eb9f57ca87b5)

**Comment by abhishekkrthakur** [comment]
https://huggingface.co/docs/autotrain/faq#is-windows-supported

**Comment by SakshiKhadilkar** [comment]
Setup this on Ubuntu 20.04.6 LTS 

$ conda create -n autotrain python=3.10
$ conda activate autotrain
$ pip install autotrain-advanced
$ conda install pytorch torchvision torchaudio pytorch-cuda=12.1 -c pytorch -c nvidia
$ conda install -c "nvidia/label/cuda-12.1.0" cuda-nvcc
$ conda install xformers -c xformers
$ python -m nltk.downloader punkt
$ pip install flash-attn --no-build-isolation # if you want to use flash-attn
$ pip install deepspeed 
$ export HF_TOKEN=your_hugging_face_write_token  (added Token )
$ autotrain --help

getting same error for " autotrain setup "

@abhishekkrthakur 

![Screenshot from 2024-05-11 12-16-34](https://github.com/huggingface/autotrain-advanced/assets/117538804/2dca71aa-8317-4225-a48c-0d89eb0968c8)

![Screenshot from 2024-05-11 12-26-23](https://github.com/huggingface/autotrain-advanced/assets/117538804/a5290552-d540-45c7-8126-83706520e2bc)

**Comment by abhishekkrthakur** [comment]
youtube tutorials are outdated. please follow docs: https://hf.co/docs/autotrain

**Comment by SakshiKhadilkar** [comment]
[https://huggingface.co/docs/autotrain/quickstart](url)

Yes, followed the same, still getting those errors
